ruby/cross_service_examples/item-tracker/README.md

```diff
@@ -0,0 +1,100 @@
+The Aurora cluster is deployed using the AWS Cloud Development Kit (AWS CDK). 
+For more information, see the [Aurora Serverless App CDK script README.md](../../../resources/cdk/aurora_serverless_app/README.md).
+
+***
+
+# Invoke this example code
+
+## Pre-requisites
+To explore this example, you must first:
+1. Create account & configure credentials using [instructions in the top-level README.md]((../../../README.md#invoke-example-code)).
+2. Install Ruby and resolve dependencies using [instructions in the Ruby README.md](../../../ruby/README.md).
+3. Create AWS infrastructure using [instructions in the Aurora Serverless App README.md](../../../resources/cdk/aurora_serverless_app/README.md). 
+    * NOTE: Store the following output values in the `helpers/config.yml` file: `CLUSTER_ARN`, `SECRET_ARN`, and `DATABASE`. Please store these in the `helpers/config.yml` file.
+4. Populate your database with a table and data using the following commands:
+     ```bash
+     cd ruby/cross_service_examples/item-tracker/helpers
+     ruby create_table.rb # checks for database and creates a new table
+     ruby populate_data.rb # creates a table with 10 records
+     ```
+5. Register your email address with Amazon Simple Email Service (SES) using the [instructions on the AWS SES docs](https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html)
+   * NOTE: Use this email for the `sender_email` and `recipient_email` fields in your `helpers/config.yml` file.
+
+## Start the application
+To view this example in its entirety, you must:
+1. Start the front-end React using the following commands:
+    ```bash
+     cd resources/clients/elwing # located within Resources sub-directory
+     npm install # first time only
+     npm start # launches a browser session on localhost:3000 
+    ```
+2. Open a second tab in your terminal and run the following command:
+    ```bash
+    cd ruby/cross_service_examples/item-tracker
+    ruby app.rb # starts REST API listening on port 8080
+    ```
+Now, visit http://localhost:3000/item_tracker to view your working application.
+
+***
+
+# Test this example code
+You may test this application manually via the front-end and automatically via the RSpec tests.
+
+### Manual validation (via front-end React app)
+Once started, you may manually validate the following behaviors on the front-end:
+* Filter items by All, Archived, and "Active" (not archived).
+* Archive an item by clicking the "Archive" button.
+* Add an item by submitting the form revealed by clicking the "Add Item" button.
+* Send a report of items by entering a verified email address and clicking "Submit".
+
+As you interact with the React app, logs will appear in the command line tab where you started the REST API. 
+### Automated validation (via RSpec tests)
+Validate the internal logic within this example by running the following commands:
+```bash
+cd spec
+rspec db_wrapper_spec.rb
+```
+
```

Comment:
Need a cleanup section for how to delete the Aurora cluster, etc.
Also additional resources section (see template).